services/npm/npm-unpacked-size.service.js

```diff
@@ -0,0 +1,65 @@
+import Joi from 'joi'
+import prettyBytes from 'pretty-bytes'
+import { pathParam } from '../index.js'
+import NpmBase, { packageNameDescription } from './npm-base.js'
+
+export default class NpmUnpackedSize extends NpmBase {
+  static category = 'size'
+
+  static route = {
+    base: 'npm/unpacked-size',
+    pattern: ':packageName/:version*',
+  }
+
+  static openApi = {
+    '/npm/unpacked-size/{packageName}': {
+      get: {
+        summary: 'NPM Unpacked Size',
+        parameters: [
+          pathParam({
+            name: 'packageName',
+            example: 'npm',
+            description: packageNameDescription,
+          }),
+        ],
+      },
+    },
+    '/npm/unpacked-size/{packageName}/{version}': {
+      get: {
+        summary: 'NPM Unpacked Size (with version)',
+        parameters: [
+          pathParam({
+            name: 'packageName',
+            example: 'npm',
+            description: packageNameDescription,
+          }),
+          pathParam({
+            name: 'version',
+            example: '4.18.2',
+          }),
+        ],
+      },
+    },
+  }
+
+  async fetch({ packageName, version }) {
+    return this._requestJson({
+      schema: Joi.any(),
+      url: `https://registry.npmjs.org/${packageName}/${version}`,
+    })
+  }
+
+  async handle({ packageName, version }) {
+    const { dist } = await this.fetch({
+      packageName,
+      version: version ?? 'latest',
+    })
+    const { unpackedSize } = dist
+
+    return {
+      label: 'unpacked size',
+      message: unpackedSize ? prettyBytes(unpackedSize) : 'unknown',
+      color: unpackedSize ? 'blue' : 'lightgray',
```

Comment:
Can you provide any detail on when this value would or would not be populated in the response?

Reply:
- [Express 4.18.2 Registry Endpoint](https://registry.npmjs.org/express/4.18.2)
 - [Express 4.15.0 Registry Endpoint](https://registry.npmjs.org/express/4.15.0)

`4.15.0` does not have `dist.unpackedSize` while `4.18.2` or `latest` has it.

I think the field is populated when someone publishes a package on NPM, so some older versions of the same package (especially when the package is old) does not have `dist.unpackedSize`.